freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/user/event/SetPasswordRequest.java

```diff
@@ -43,6 +50,7 @@ public String toString() {
                 + "stackId='" + getResourceId() + '\''
                 + "environment='" + environment + '\''
                 + "username='" + username + '\''
+                + "userCrn='" + userCrn + '\''
```

Comment:
So not you, but there are nicer ways to do this, like using guava object helper.

Reply:
Ah nice one. I'll try it out on new classes. And I'll keep this one the same for consistency.